### PR TITLE
add collections rss feed

### PR DESCRIPTION
**Pull request by lbiedinger**

### CODE REVIEW

File: app/views/collections/index.rss.builder [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    xml.lastBuildDate(DateTime.now.rfc2822)
+    xml.tag!('atom:link', rel: 'self', type: 'application/rss+xml', href: collections_url(format: 'rss'))
+
+    displayable_collections.each do |collection|
+      xml.item do
+        xml.title(collection.title)
+        xml.link(collection_url(collection))
+        xml.pubDate(collection.updated_at.rfc2822)
+        xml.guid(collection_url(collection))
+        xml.description(collection.landing_page.strapline)
```
Comment (rwd) [inline app/views/collections/index.rss.builder]: This needs to populate the `%{total_item_count}` placeholder.

File: app/views/collections/index.rss.builder [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# Helper method for the collections description
+# derived from the associated landing page strapline.
+def strapline(collection)
```
Comment (rwd) [inline app/views/collections/index.rss.builder]: Let's move this into `CollectionsHelper` as `collection_strapline`, then make both this RSS view and the `Collections::Show` Mustache view class use it, to keep it DRY, removing the assignment of `@total_item_count` from `CollectionsController#show`.

File: spec/controllers/collections_controller_spec.rb [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
+      end
+
+      it 'redirects to home' do
+        expect(response).to redirect_to(home_url)
+      end
     end
 
-    it 'redirects to home' do
-      expect(response).to redirect_to(home_url)
+    context 'when requesting as an rss feed' do
```
Comment (rwd) [inline spec/controllers/collections_controller_spec.rb]: Could we spec that only published collections are included?

Comment (lbiedinger) [reply spec/controllers/collections_controller_spec.rb]: Added feature spec for the  collections rss feed that includes these checks

**Pull request merged**